src/pkl/index.pkl

```diff
@@ -16,7 +16,7 @@
 
 output {
   files {
-    ["out/pkl.tmLanguage.json"] = import("pkl.tmLanguage.pkl").output
+    ["grammars/pkl.tmLanguage.json"] = import("pkl.tmLanguage.pkl").output
     ["out/pkl.snippets.json"] = import("snippets.pkl").output
```

Comment:
if separating out by category, snippets still going to `out` seems awkward naming.

Reply:
Thought about that... We could create a `snippets` folder in the root.